src/dotnet/Users.Contracts/IUserProfiles.cs

```diff
@@ -4,4 +4,9 @@ public interface IUserProfiles
 {
     [ComputeMethod(KeepAliveTime = 10)]
     Task<UserProfile?> Get(Session session, CancellationToken cancellationToken);
+
+    [CommandHandler]
+    public Task UpdateStatus(UpdateStatusCommand command, CancellationToken ct = default);
```

Comment:
`ct` -> `cancellationToken`

Reply:
A note on `CancellationToken cancellationToken = default`: we've agreed to avoid `default` for our own APIs, i.e. the only possible place to have `default` is if you assume that you're writing a library code that might be later moved to some dedicated project & open-sourced.

The reason for this is kinda simple: it's almost always a mistake to skip on passing `cancellationToken`.

Reply:
done